src/libs/actions/User.js

```diff
@@ -541,7 +541,7 @@ function subscribeToUserEvents() {
 
 /**
  * Sync preferredSkinTone with Onyx and Server
- * @param {String} skinTone
+ * @param {Number} skinTone
  */
 function updatePreferredSkinTone(skinTone) {
     const optimisticData = [
```

Comment:
This `updatePreferredSkinTone` is used in 2 places

https://github.com/Expensify/App/blob/502df9bc99accb7ac82b703d1105d18d34fe880f/src/components/EmojiPicker/EmojiPickerMenu/index.js#L408C1-L419C3

and

https://github.com/Expensify/App/blob/502df9bc99accb7ac82b703d1105d18d34fe880f/src/components/EmojiPicker/EmojiPickerMenu/index.native.js#L89C6-L98

In both of the usages of this `updatePreferredSkinTone` function, you can see that their JSDoc suggests that skinTone is `Number` (and not `String`)

Reply:
It is indeed an int.